feat(forms): add `Value.Selection` component (#3857)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection/info.mdx

---
showTabs: true
---

## Description

> `Value.Selection` is a wrapper component for displaying string values, with user experience tailored for selection values.

Reviewer: ```suggestion
`Value.Selection` is a component for displaying a string value based on a user selection.
```
